stl/inc/experimental/random

```diff
@@ -0,0 +1,66 @@
+// random experimental header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _EXPERIMENTAL_RANDOM_
+#define _EXPERIMENTAL_RANDOM_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#include <random>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace experimental {
+    inline namespace fundamentals_v3 {
+
+        _NODISCARD inline default_random_engine& _Engine() {
+            thread_local default_random_engine _Eng{random_device{}()};

```

Comment:
This is still being seeded by a 32-bit seed, `random_device{}()`, so it's affected by seed collisions at thread startup. Your zero-argument `reseed()` below should be refactored, so that `_Eng` construction can share this code.